src/rust/bitbox02/src/secp256k1.rs

```diff
@@ -58,19 +72,32 @@ pub fn _secp256k1_nonce_commit(
     msg: &[u8; 32],
     host_commitment: &[u8; 32],
 ) -> Result<[u8; EC_PUBLIC_KEY_LEN], ()> {
-    let mut signer_commitment = [0u8; EC_PUBLIC_KEY_LEN];
-    match unsafe {
-        bitbox02_sys::keystore_secp256k1_nonce_commit(
+    let mut signer_commitment = MaybeUninit::<bitbox02_sys::secp256k1_ecdsa_s2c_opening>::uninit();
```

Comment:
why do you prefer uninitialized bytes here instead of `zeroed()` (like earlier)?